tpcc/main.go

```diff
@@ -43,6 +43,7 @@ var loadIndexes = flag.Bool("load-indexes", false, "Load indexes. Implied by loa
 var maxOps = flag.Uint64("max-ops", 0, "Maximum number of operations to run")
 var noWait = flag.Bool("no-wait", false, "Run in no wait mode (no think/keying time)")
 var opsStats = flag.Bool("ops-stats", false, "Print stats for all operations, not just newOrders")
+var rampMs = flag.Uint64("rampMs", 10000, "Number of milliseconds to ramp up workers over.")
```

Comment:
Does this really need to be milliseconds? Also, you can make this a `flag.Duration`. Lastly, `rampMs` is inconsistent capitalization with the other flags.